.pre-commit-config.yaml

```
@@ -11,13 +11,13 @@
 repos:
   # Autoformat: markdown, yaml
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

```suggestion
    rev: v3.1.0
```

I've seen an issues with 4.0.0-alpha that I've not figured out to resolve yet, lets manually hold back